src/main/java/com/github/wovnio/wovnjava/HtmlConverter.java

```diff
@@ -179,6 +184,20 @@ private void appendHrefLang() {
         }
     }
 
+    private void translateCanonicalTag(String lang) {
+        Elements elements = doc.head().getElementsByTag("link");
```

Comment:
jsoup allows selection by CSS tag like so `Elements links = doc.select("a[rel=\"canonical\"]")` could be simpler.

https://jsoup.org/cookbook/extracting-data/selector-syntax